### PR TITLE
Sync with upstream

### PR DESCRIPTION
**Pull request by honza**
* Add redux-form (MIT)
* Remove karma, jasmine, and phantomjs

### CODE REVIEW

**Comment by honza** [comment]
@apevec @hguemar

File: openstack-tripleo-ui-deps.spec [hunk 1]
```diff
@@ -25,7 +25,6 @@ ExclusiveArch: x86_64
 BuildRequires:  nodejs
 BuildRequires:  git
 Requires:       %{sname}-babel = %{version}-%{release}
-Requires:       phantomjs >= 1.9.7
```
Comment (apevec) [inline openstack-tripleo-ui-deps.spec:28]: How come this dep is gone, can you link to upstream where it was removed?

File: openstack-tripleo-ui-deps.spec [hunk 2]
```diff
@@ -65,15 +63,21 @@ cp -rf %{_builddir}/node_modules %{buildroot}/opt/%{name}/
 /opt/%{name}/node_modules/webpack*
 
 %changelog
+
+* Wed Mar 22 2017 Honza Pokorny <honza@redhat.com> 7-2
```
Comment (apevec) [inline openstack-tripleo-ui-deps.spec]: Release: field was not bumped to 2

File: openstack-tripleo-ui-deps.spec [hunk 3]
```diff
@@ -65,15 +63,21 @@ cp -rf %{_builddir}/node_modules %{buildroot}/opt/%{name}/
 /opt/%{name}/node_modules/webpack*
 
 %changelog
+
```
Comment (apevec) [inline openstack-tripleo-ui-deps.spec]: nitpick: please remote this empty line

**Review by apevec — CHANGES_REQUESTED** [review]

File: openstack-tripleo-ui-deps.spec (hunk 1 above)
Comment (honza) [reply openstack-tripleo-ui-deps.spec:28]: It was removed in [this commit](https://github.com/openstack/tripleo-ui/commit/0376dfe55f487abcec517ea1ec512f58a5bd27ec).

**Review by apevec — APPROVED** [review]

**Pull request merged**